### PR TITLE
New package Jackett

### PR DESCRIPTION
**Pull request by PF4Public**
I hope @candrews won't mind keeping an eye on this package

### CODE REVIEW

File: acct-user/jackett/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project" proxied="proxy">
```
Comment (thesamesam) [inline acct-user/jackett/metadata.xml:4]: This needs a human along with it.

File: acct-user/jackett/jackett-0-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2020-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-user
+
+DESCRIPTION="User for jackett"
+ACCT_USER_HOME=/var/lib/jackett
+ACCT_USER_ID=531
```
Comment (thesamesam) [inline acct-user/jackett/jackett-0-r1.ebuild:10]: Please use -1 when submitting a PR, then the merger can allocate one.

File: acct-user/jackett/metadata.xml (hunk 1 above)
Comment (PF4Public) [reply acct-user/jackett/metadata.xml:4]: I can put myself if that matters.

File: acct-user/jackett/jackett-0-r1.ebuild (hunk 2 above)
Comment (PF4Public) [reply acct-user/jackett/jackett-0-r1.ebuild:10]: Sorry, I didn't know that.

**Comment by candrews** [comment]
Thank you!

File: acct-user/jackett/jackett-0-r1.ebuild (hunk 2 above)
Comment (candrews) [reply acct-user/jackett/jackett-0-r1.ebuild:10]: I took care of making sure the UID/GID was available and reserving it.

But in the future, `-1` is the suggested approach.

Thanks again!

**Pull request closed without merging**